apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,40 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I can mark a book as finished', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('c++');
+    await form.submit();
+
+    // Add Book to List
+    const items = await $$('[data-testing="book-item"]').first();
+    await items.$$('[data-testing="add-book"]').first().click();
+
+    // Open list
+    const readingListToggle = await $('[data-testing="toggle-reading-list"]');
+    await readingListToggle.click();
+
+    // Mark Book as Finished
+    const markFinishedButton = await $$('[data-testing="reading-list-content"]').first().getWebElement().findElement(By.css('.reading-list-item--details--complete-button'));
```

Comment:
This is an obscenely long line. Are you sure this is as readable as it could be?

As with the previous PR, do we really expect more than one `reading-list-content`?

Reply:
Made the appropriate change